<!DOCTYPE html>
<!--[if lt IE 7]>	  <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>		 <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>		 <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
	<head>
		<meta charset="utf-8">
		<!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"><![endif]-->
		<title>Jungbluth Lab, Sean</title>
		<meta name="keywords" content="Oceanography, Biology, Molecular Ecology, Genetics, Bioinformatics" />
		<meta name="description" content="Welcome to the homepage for Sean Jungbluth, PhD. Sean is a microbial ecologist, computational biologist, and oceanographer exploring the deep subseafloor and Earth's unique microbial life." />
		<meta name="viewport" content="width=device-width">
		<link href='https://fonts.googleapis.com/css?family=Raleway:400,600,500,300,700' rel='stylesheet' type='text/css'>
		<link rel="stylesheet" href="css/bootstrap.min.css">
		<link rel="stylesheet" href="css/templatemo_main.css">
		<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
		<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
		ga('create', 'UA-39206053-4', 'auto');
		ga('require', 'linkid');
		ga('send', 'pageview');
		</script>
		<style>
		#map_wrapper {
			height: 400px;
		}
		#map_canvas {
			width: 100%;
			height: 100%;
		}
		</style>
	</head>
	<body>
		<script src="js/jquery.min.js"></script>
		<script src="js/jquery.backstretch.min.js"></script>
		<script src="js/templatemo_script.js"></script>
		<script src="js/google-maps.js"></script>
		<script src="js/jssor.slider.mini.js"></script>
		<script src="js/embed.js"></script>
		<div id="main-wrapper">
			<!--[if lt IE 7]>
				<p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a rel="nofollow" href="http://browsehappy.com">upgrade your browser</a> or <a rel="nofollow" href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
			<![endif]-->
			<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 affix text-center" style="z-index: 1;">
				<h1 class="templatemo-site-title">
						<a href="#"></a>
						<img src="images/btn-menu.png" alt="main menu" id="m-btn" class="pull-right visible-xs visible-sm" >
				</h1>
				<ul id="responsive" style="display:none" class="hidden-lg hidden-md"></ul><!-- /.responsive -->
			</div>
			<div class="menu visible-md visible-lg">
				<ul id="menu-list">
					<li class="active home-menu"><a href="#home">About Me</a></li>
					<li class="vita-menu"><a href="#vita">--> Curriculum Vitae</a></li>
 					<li class="consulting-menu"><a href="#consulting">--> Consulting</a></li>
					<li class="research-menu"><a href="#research">Research</a></li>
					<li class="publications-menu"><a href="#publications">--> Publications</a></li>
					<li class="softwarepublished-menu"><a href="#softwarepublished">--> Software</a></li>
					<li class="data-menu"><a href="#data">--> Data</a></li>
<!-- 					<li class="field-menu"><a href="#field">Oceanographic and Field Experience</a></li>
 -->
<!--  				<li class="fun-menu"><a href="#fun">Fun Science Stuff</a></li>
 --><!-- 					<li class="portfolio-menu"><a href="#portfolio">Image Portfolio</a></li>
 -->
<!-- 					<li class="news-menu"><a href="#news">News</a></li>
 -->
					<li class="main-menu"><a href="#main">Jungbluth Lab</a></li>

				</ul>
			</div><!-- /.menu -->
			<div class="image-section">
				<div class="image-container">
					<img src="images/DSC_0063.jpg" id="home-img" class="main-img inactive" alt="Home">
					<img src="images/DSC_0341.jpg" id="vita-img" class="inactive" alt="vita">
					<img src="images/DSC_0407.jpg" id="research-img"  class="inactive" alt="research">
					<img src="images/IMG_0229.jpg" id="publications-img" class="inactive" alt="publications">
					<img src="images/DSC_0341.jpg" id="field-img" class="inactive" alt="field">
					<img src="images/pexels-photo-247791_mod.png" id="data-img" class="inactive" alt="data">
					<img src="images/IMG_0136.jpg" id="fun-img" class="inactive" alt="fun">
					<img src="images/IMG_0088.jpg" id="uc-img" class="inactive" alt="uc">
					<img src="images/2008_07_30_19_55_25.jpg" id="portfolio-img" class="inactive" alt="portfolio">
					<img src="images/DSC_0063.jpg" id="consulting-img" class="inactive" alt="consulting">
					<img src="images/DSC_0974.jpg" id="softwarepublished-img" class="inactive" alt="softwarepublished">
					<img src="images/IMG_0111.jpg" id="news-img" class="inactive" alt="news">
					<img src="images/IMG_0088.jpg" id="contact-img" class="inactive" alt="contact">
					<img src="images/DSC_0425.jpg" id="main-img" class="inactive" alt="main">
				</div>
			</div>
			<div class="row">
				<div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 col-lg-offset-6 col-md-offset-6 templatemo-content-wrapper">
					<div class="templatemo-content">
						<section id="home-text" class="active templatemo-content-section">
							<div class="col-sm-12 col-md-12 col-lg-12">
								<h2></h2>
								<p>
									<a target="_blank" href="https://github.com/jungbluth">Github</a>&nbsp;&nbsp;&nbsp;&nbsp;
									<a target="_blank" href="https://www.linkedin.com/in/sean-jungbluth-2230b314">LinkedIn</a>&nbsp;&nbsp;&nbsp;&nbsp;
									<a target="_blank" href="https://scholar.google.com/citations?user=2h4vdZgAAAAJ&hl=en">Google Scholar</a>&nbsp;&nbsp;&nbsp;&nbsp;
									<a target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Jungbluth+SP">PubMed</a>&nbsp;&nbsp;&nbsp;&nbsp;
									<a target="_blank" href="http://orcid.org/0000-0001-9265-8341">ORCID</a>&nbsp;&nbsp;&nbsp;&nbsp;
<!-- 									<a target="_blank" href="https://www.researchgate.net/profile/Sean_Jungbluth">ResearchGate</a>&nbsp;&nbsp;&nbsp;&nbsp;
									<a target="_blank" href="https://www.mendeley.com/profiles/sean-jungbluth/">Mendeley</a>&nbsp;&nbsp;&nbsp;&nbsp;
									<a target="_blank" href="http://www.researcherid.com/rid/A-9054-2012">ResearcherID</a>&nbsp;&nbsp;&nbsp;&nbsp; -->
								</p>
								<h1>My Passion</h1>
								<h2></h2>
								<p>Exploration of Earth's microbial and viral diversity is a major scientific frontier. Sequencing DNA - the blueprint of life - collected directly from the environment provides a powerful way to access the fantastic microbial diversity on our planet. These mostly unknown lifeforms are ripe with novel functions, metabolisms, the potential for biotechnological and medical applications. They will also inform us about the evolution of all life.</p>
								<h2></h2>
								<p>My previous work has explored the nature and extent of microbial life underground and below the seafloor using tools rooted in molecular biology, biogeochemistry, and microbiology. Now I employ computational tools and leverage big data to continue exploring microbes at a larger scale. And while doing so, I develop software enabling molecular microbial ecology research for others.</p>
								<h2></h2>
								<p>Check out my work featured in popular media outlets!</p>
								<p>
								<h1>Contact</h1>
								<h2></h2>
								<p> jungbluth dot sean at gmail dot com</p>
								<a target="_blank" href="https://www.facebook.com/nytimesscience/videos/1111185458961479/"><img src="images/The_New_York_Times_logo.png" width="250" height="31" border="5"></a>&nbsp;&nbsp;&nbsp;&nbsp;
								<a target="_blank" href="http://voices.nationalgeographic.com/2016/10/22/life-on-the-shore-based-team-the-other-half-of-a-telepresence-expedition/"><img src="images/National-Geographic-logo.png" width="125" height="90" border="5"></a>&nbsp;&nbsp;&nbsp;&nbsp;
								<a target="_blank" href="http://www.latimes.com/science/sciencenow/la-sci-sn-microbe-undersea-aquifer-20150114-story.html"><img src="images/2000px-Los_Angeles_Times_logo.svg.png" width="250" height="31" border="5"></a>&nbsp;&nbsp;&nbsp;&nbsp;
								<a target="_blank" href="https://astrobiology.nasa.gov/news/life-in-low-temperature-fluids-beneath-the-ocean-crust/"><img src="images/nic_print1500x1247ppi.png" width="75" height="60" border="5"></a>&nbsp;&nbsp;&nbsp;&nbsp;
								<a target="_blank" href="https://www.sciencedaily.com/releases/2015/01/150115091910.htm"><img src="images/scidaily-logo1.png" width="200" height="40" border="5"></a>&nbsp;&nbsp;&nbsp;&nbsp;
								<a target="_blank" href="http://www.scienceworldreport.com/articles/21285/20150115/new-species-discovered-two-miles-below-surface-ocean.htm"><img src="images/logo-scienceworldreport.png" width="200" height="25" border="5"></a>&nbsp;&nbsp;&nbsp;&nbsp;
								<a target="_blank" href="http://oceans.mit.edu/news/featured-stories/new-york-times-follows-young-ocean-scientists-training-in-new-england-waters"><img src="images/MIT_logo.svg.png" width="80" height="41" border="5"></a>&nbsp;&nbsp;&nbsp;&nbsp;
								<a target="_blank" href="http://news.usc.edu/73851/new-species-discovered-beneath-ocean-crust/"><img src="images/Interlocking_USC_Logo.svg.png" width="44" height="60" border="5"></a>&nbsp;&nbsp;&nbsp;&nbsp;
								<a target="_blank" href="http://manoa.hawaii.edu/news/article.php?aId=6987"><img src="images/Uh_Manoa_Logo_04.png" width="70" height="52" border="5"></a>&nbsp;&nbsp;&nbsp;&nbsp;
								</p><h2></h2>
<!-- 								<p>For a great overview of my deep biosphere research, check out a seminar I delivered as an invited participant of the C-DEBI Networked Speaker seminar series: <a rel="nofollow" href="http://www.darkenergybiosphere.org/outputs-resources/networked-speaker-series/nss-sean-jungbluth/" target="_blank">Link to info about video</a></p><h2></h2>
								<div class="sites-embed-content sites-embed-type-youtube"><iframe title="YouTube video player" class="youtube-player" type="text/html" src="//www.youtube.com/embed/n30C6dMKgZc?rel=0&amp;wmode=opaque" frameborder="0" allowFullScreen="true" width="600" height="400"></iframe></div><h2></h2> -->
<!-- 								<h3>Twitter: @seanjungbluth</h3>
								<a class="twitter-timeline" width="500" height="75" href="https://twitter.com/seanjungbluth" data-chrome="noheader noborders nofooter noscrollbar transparent" data-tweet-limit="2" data-widget-id="458477274694578177">Tweets by @seanjungbluth</a>
								<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
 -->							</div>
						</section><!-- /.home-text -->
						<section id="vita-text" class="inactive">
							<div class="col-sm-12 col-md-12 col-lg-12">
								<h1>Curriculum Vitae</h1>
								<h2></h2>
								<p><a rel="nofollow" href="docs/Jungbluth_CV_071419_short_coding.pdf" target="_blank"><b>Programming (4 pages)</b></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a rel="nofollow" href="docs/Jungbluth_CV_071419_short.pdf" target="_blank"><b>Academic, Short (4 pages)</b></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a rel="nofollow" href="docs/Jungbluth_CV_071419.pdf" target="_blank"><b>Academic, Long (20 pages)</b></a></p>
								<h2></h2>
								<h2>Professional Appointments</h2>
								<p>2019-present, Data Scientist, Lawrence Berkeley National Laboratory, Department of Environmental Genomics & Systems Biology, Berkeley, CA</p>
								<p>2019-present, Affiliate Scientist, University of California, Department of Bioengineering, Berkeley, CA</p>
								<p>2017-2019, Computational Biologist Fellow, Lawrence Berkeley National Laboratory, Joint Genome Institute, Walnut Creek, CA</p>
								<p>2016-2017, Independent Scientist, Honolulu, HI</p>
								<p>2015-2016, Research Associate, University of Southern California, Department of Earth Sciences, Honolulu, HI</p>
								<p>2014-2015, Research Scholar, University of Hawaii, Department of Oceanography, Honolulu, HI</p>
								<h2>Education</h2>
								<p>2014&nbsp;&nbsp;PhD&nbsp;&nbsp;Oceanography, University of Hawaii</p>
								<p>2007&nbsp;&nbsp;BS&nbsp;&nbsp;&nbsp;Biology and Bacteriology, University of Wisconsin</p>
								<h2>Selected Awards and Honors</h2>
								<p>U.S. Department of Energy Joint Genome Institute Computational Biology Postdoctoral Fellowship, Recipient</p>
								<p>University-National Oceanographic Laboratory System Deep-Submergence Science Leadership Cruise, Invited Participant</p>
								<p>Deep Carbon Observatory Origins and Movements of Subsurface Microorganisms Workshop, Invited Participant</p>
								<p>Consortium for Ocean Leadership U.S. Science Support Program Marine Geoscience Leadership Symposium, Invited Participant</p>
								<p>National Science Foundation Ecological Dissertations in the Aquatic Sciences Symposium, Invited Participant</p>
								<p>Center for Dark Energy Biosphere Investigations Graduate Student Fellowship, Recipient</p>
								<p>Consortium for Ocean Leadership U.S. Science Support Program Schlanger Ocean Drilling Fellowship, Recipient</p>
								<h2>Teaching</h2>
								<p>Fundamentals of Oceanography<br>Fundamentals of Microbiology</p>
								<h2>Selected Presentations</h2>
								<p>Jungbluth SP, Kirton E, Roux S, Wu D, Nielsen T, Ivanova NN, Kyrpides NC, Eloe-Fadrosh EA (2017, July) Illuminating the phylogenetic breadth and metabolic potential of microbial life with thousands of archaeal metagenome-assembled genomes. Archaea: Ecology, Metabolism & Molecular Biology – <b>Gordon Research Conference</b>, Waterville Valley, NH.</p>
								<p>Rappe MS, Jungbluth SP, Lin H-T, Nigro OD, Steward GF (2016, March) The distribution, diversity, and function of microbial and viral communities inhabiting anoxic crustal fluids within deep subseafloor basalt. Presented at the <b>Joint Genome Institute</b> Users Meeting, Walnut Creek, CA.</p>
								<p>Jungbluth SP, Glavina del Rio T, Tringe S, Stepanauskas R, Rappe MS (2015, December) Ecogenomic characterization of a marine microorganism belonging to a Firmicutes lineage that is widespread in both terrestrial and oceanic subsurface environments. Presented at the fall meeting of the <b>American Geophysical Union</b>, San Francisco, CA.</p>
								<p>Nigro O, Jungbluth SP, Lin H-T, Rappe MS, Steward G (2015, December) Archaeal viruses contribute to the novel viral assemblage inhabiting oceanic, basalt-hosted deep subsurface crustal fluids. Presented at the fall meeting of the <b>American Geophysical Union</b>, San Francisco, CA.</p>
								<p>Nigro O, Jungbluth SP, Lin H-T, Hsieh C-C, Mueller J, Schvarcz C, Rappe MS, Steward G (2015, February) A first look at a deep subsurface viral community from the basement fluids of the Juan de Fuca Ridge flank. Presented at the <b>American Society for Limnology and Oceanography</b> Aquatic Sciences Meeting, Grenada, Spain.</p>
								<p>Lloyd KG, Bird J, Steen AD, Buongiorno J, Jungbluth SP (2015, January) Insights from the first deep subsurface single cell genomes and development of digestR, a new tool for metagenomic analysis. Presented at the <b>Center for Dark Energy Biosphere Investigations</b> NSF Site Visit, Los Angeles, CA.</p>
								<p>Jungbluth SP, Lin H-T, Hsieh C-C, Rappe MS (2014, December) High-resolution temporal analysis of deep subseafloor microbial communities inhabiting basement fluids. Presented at the fall meeting of the <b>American Geophysical Union</b>, San Francisco, CA.</p>
								<p>Carr S, Jungbluth SP, Lin H-T, Rappe MS (2014, December) Investigating chemotactic potential within crustal fluid communities. Presented at the fall meeting of the <b>American Geophysical Union</b>, San Francisco, CA.</p>
								<p>Orcutt BN, Sylvan J, Rogers D, Lee R, Girguis P, Carr S, Jungbluth SP, Rappe MS (2014, December) Carbon fixation in oceanic crust: does it happen, and is it important?. Presented at the fall meeting of the <b>American Geophysical Union</b>, San Francisco, CA. (Invited Presentation)</p>
								<p>Jungbluth SP, Bowers R, Lin, H-T, Hsieh C-C, Cowen JP, Rappe MS (2012, December) Microbial diversity within Juan de Fuca ridge basement fluids sampled from oceanic borehole observatories. Presented at the fall meeting of the <b>American Geophysical Union</b>, San Francisco, CA. (Invited Presentation)</p>
								<h2>Invited Seminars</h2>
								<p>Jungbluth SP (2019, January) Microbial genomics of marine and terrestrial biomes and opportunities for collaborative science. Seminar presented to the Systems Biology Knowledge Base (KBase). <b>Lawrence Berkeley National Laboratory</b>, Berkeley, California</p>
								<p>Jungbluth SP (2016, January) Investigating novel microbial life in the deep igneous oceanic crust. Seminar presented at the Department of Earth Sciences Paleoenvironmental Seminar Series. <b>University of Southern California</b>, Los Angeles, California</p>
								<p>Jungbluth SP (2015, November) Microbial life in the deep subseafloor igneous basement: a frontier of exploration. Seminar presented at the Marine Environmental Biology Department Retreat. <b>USC Wrigley Institute for Environmental Studies</b>, Catalina Island, California</p>
								<p>Jungbluth SP (2014, October) Juan de Fuca Ridge flank basement fluid microbiology. Seminar presented as part of the Center for Dark Energy Biosphere Investigations Networked Speaker Seminar Series. <b>[Broadcast Online]</b></p>
								<p>Jungbluth SP (2014, October) Microbial diversity in anoxic, subseafloor basaltic fluids. Seminar presented at the Astrobiology Seminar Series. <b>University of Hawaii</b>, Honolulu, Hawaii</p>
								<h2>Professional Activities</h2>
								<p>Member of: American Society for Microbiology, Association for the Sciences of Limnology and Oceanography, American Association for the Advancement of Science, American Geophysical Union, Astrobiology Society, International Society for Microbial Ecology, International Society for Subsurface Microbiology, The Oceanography Society </p>
								<p>Reviewer for: Environmental Microbiology, Frontiers in Microbiology, The ISME Journal, National Science Foundation (Biological Oceanography), PLoS ONE</p>
							</div>
						</section><!-- /.vita-text -->
						<section id="research-text" class="inactive">
							<h2 class="text-center">My research</h1>
							<div class="col-sm-12 col-md-12 col-lg-12">
								<h2>Exploration of the Deep Biosphere </h2>
								<p><b>Summary:</b> For over seven years I worked on a team that was tasked with exploration of the deep subseafloor biosphere using manned and unmanned deep submersibles and ocean drilling program boreholes. In the deep-sea, we sampled fluids filling the spaces and microfractures in the basaltic rock using hundred meter long pipes installed into the seafloor. This technology provided an access point to the remote deep ocean volcanic rock and the extreme microbial lifeforms that live there. </p>
								<center><img src="images/InfoGraphic_CorkExplainer-01.jpg" alt="CORK-infographic" width="415" height="555" border="5"></center>
								<p><b>Above:</b> Infographic describing how we access microorganisms in the deep subseafloor. <a rel="nofollow" href="images/InfoGraphic_CorkExplainer.pdf" target="_blank">Full-size Version (6.7 MB)</a></p>
								<h2>Genomics of Subsurface Bacteria and Archaea</h2>
								<p><b>Summary:</b> The genomic repertoire of bacteria and archaea inhabiting the subsurface are revealing clues to the adaptations required for life to survive in an environment devoid of sunlight. Cutting-edge DNA sequencing techniques and bioinformatics tools allow for recovery of near-complete genomes from environmental samples and are allowing for unprecedented investigations into the genomic potential of microbial communities. </p>
								<center><img src="images/genomics.jpg" alt="genomics" width="500" height="340" border="5"></center>
								<p><b>Above:</b> Genomic comparisons reveal clues to the metabolic features and adaptations of deep subsurface life.</p>
								<h2>Diversity of Microorganisms Inhabiting Igneous Rock</h2>
								<p><b>Summary:</b> The diversity of microbial life inhabiting the rocky parts of Earth are poorly described. Microbes living on and inside rocks slowly eat the minerals and participate in the weathering of the Earth on global scales. Understanding the diversity of Bacteria and Archaea in the subsurface of the global ocean and basins and the role of this diversity in establishing community functioning is of high interest to me. </p>
								<center><img src="images/microbial-div.jpg" alt="microbial-div" width="500" height="340" border="5"></center>
								<p><b>Above:</b> Understanding the abundance and diversity of microbes in the environment helps to understand the hidden functions that occur there.</p>
								<h2>Viruses in the Deep Subsurface</h2>
								<p><b>Summary:</b> The diversity of viruses on the planet is among the least catalogued of all life. I have been working to describe the diversity of novel viruses in the deep marine subseafloor and explore their genomic composition and potential for interaction with the other domains of life.</p>
								<center><img src="images/viruses.jpg" alt="viruses" width="500" height="340" border="5"></center>
								<p><b>Above:</b> Viral diversity investigations enabled by viral metagenomics are revealing novel thermophilic viruses. Analysis of microbial CRISPR sequences and spacers allows for understanding of virus-host dynamics in the environment. </p>
								<h2>Professional Development for Scientists</h2>
								<p><b>Summary:</b> In addition to research, I believe it is important to promote diverse professional development opportuntities for young scientists. There are more professionally-trained scientists than academic positions allow, so additional useful modes of employment must be made available to trainees. </p>
								<center><img src="images/prof-dev.jpg" alt="prof-dev" width="500" height="340" border="5"></center>
								<p><b>Above:</b> Trainees are facing a harsh employment situation upon graduation and networking is key to success. </p>
							</div>
						</section><!-- /.research-text -->
						<section id="portfolio-text" class="inactive">
							<h2 class="text-center">Image Portfolio</h1>
								<h2></h2>
							<div class="col-sm-12 col-md-12 col-lg-12">
								<p> Science communication relies on the art of figure design. Here are some example figures I've designed for research publications - please enjoy!
								<center><img src="images/1362A_CRISPR_scaffold_mapping.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/ECODAS-Microbes_Figure1-01.jpg" alt="" width="450" height="700" border="5"></center><h1></h1>
								<center><img src="images/ECODAS-Microbes_Figure2-01.jpg" alt="" width="450" height="302" border="5"></center><h1></h1>
								<center><img src="images/Fig1-Desulforudis-Phylogenome-New-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/Fig1-Desulforudis-Phylogenome-New1-01.jpg" alt="" width="330" height="302" border="5"></center><h1></h1>
								<center><img src="images/Fig2_Circos-and-COG-Comparisons-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/Fig3_GlobalMap-Connectivitiy-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/FigS2_16S-tree-01.jpg" alt="" width="450" height="400" border="5"></center><h1></h1>
								<center><img src="images/FigS3_Genome-Coding-Density-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/FigS4_Circos_SAG-mapping-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/FigS5_InDel-Examples-01.jpg" alt="" width="550" height="700" border="5"></center><h1></h1>
								<center><img src="images/FigXX_All-Gene_Phylogenetic-Diversity-01.jpg" alt="" width="500" height="302" border="5"></center><h1></h1>
								<center><img src="images/Fig_1_Chemical_and_Sample_Variation-01.jpg" alt="" width="300" height="500" border="5"></center><h1></h1>
								<center><img src="images/Fig_2_Bin-Overview_Functions-01.jpg" alt="" width="450" height="700" border="5"></center><h1></h1>
								<center><img src="images/Fig_2_Cell_count_and_Environment_Comparison-01.jpg" alt="" width="300" height="700" border="5"></center><h1></h1>
								<center><img src="images/Fig_2_Cell_count_and_Environment_Comparison1-01.jpg" alt="" width="450" height="500" border="5"></center><h1></h1>
								<center><img src="images/Fig_2_Cell_count_and_Environment_Comparison2-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/Fig_3_COG-Heirarchical-Clustering-01.jpg" alt="" width="450" height="302" border="5"></center><h1></h1>
								<center><img src="images/Fig_3_Phylum_Taxonomy_Plots-01.jpg" alt="" width="450" height="302" border="5"></center><h1></h1>
								<center><img src="images/Fig_4_HeatMap_and_Indicator_Taxa-01.jpg" alt="" width="450" height="302" border="5"></center><h1></h1>
								<center><img src="images/Fig_4_HeatMap_and_Indicator_Taxa1-01.jpg" alt="" width="550" height="302" border="5"></center><h1></h1>
								<center><img src="images/Fig_5_Phylogenetic_Analysis-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/Fig_S10_ESOM-Bins-01.jpg" alt="" width="550" height="302" border="5"></center><h1></h1>
								<center><img src="images/Fig_S15_Tetranucleotide-PCA-Combined-01.jpg" alt="" width="600" height="500" border="5"></center><h1></h1>
								<center><img src="images/Fig_S18_16S-similarity-matrix-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/Fig_S19X_Archaeoglobi-AAI-sharedgene-combined-01.jpg" alt="" width="450" height="500" border="5"></center><h1></h1>
								<center><img src="images/Fig_S20X_Heatmap-overview_v2-01.jpg" alt="" width="500" height="700" border="5"></center><h1></h1>
								<center><img src="images/Fig_S2_Bin-Overview_QC-01.jpg" alt="" width="450" height="700" border="5"></center><h1></h1>
								<center><img src="images/Fig_S2_Illumina_Flowchart-01.jpg" alt="" width="450" height="700" border="5"></center><h1></h1>
								<center><img src="images/Fig_S3_AlphaDiversity-01.jpg" alt="" width="600" height="400" border="5"></center><h1></h1>
								<center><img src="images/Fig_XX_Read-Extension-Flowchart-01.jpg" alt="" width="450" height="700" border="5"></center><h1></h1>
								<center><img src="images/Fig_XX_Read-Extension-Scaffold-Length-01.jpg" alt="" width="450" height="302" border="5"></center><h1></h1>
								<center><img src="images/Ice-Cave-16S-Overview-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/PICRUSt-final-01.jpg" alt="" width="550" height="302" border="5"></center><h1></h1>
								<center><img src="images/SURF-16S_v2-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/SURF-phylogenome-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/bin-overview-compiled_v6.jpg" alt="" width="550" height="700" border="5"></center><h1></h1>
								<center><img src="images/fig09_PCA-Map-01.jpg" alt="" width="600" height="400" border="5"></center><h1></h1>
								<center><img src="images/primer-plot-01.jpg" alt="" width="450" height="450" border="5"></center><h1></h1>
								<center><img src="images/taxonomy_Archaea-class_top25_SSF-01.jpg" alt="" width="600" height="100" border="5"></center><h1></h1>
								<center><img src="images/taxonomy_Bacteria-order_top20_SSF-01.jpg" alt="" width="600" height="100" border="5"></center><h1></h1>
							</div>
						</section><!-- /.portfolio-text -->

						<section id="main-text" class="active inactive">
							<div class="col-sm-12 col-md-12 col-lg-12">
								<h1><a href="index.html">Jungbluth Lab home</a></h1>
							</div>
						</section><!-- /.main-text -->

						<section id="publications-text" class="inactive">
							<div class="col-sm-12 col-md-12 col-lg-12">
								<h1>Publications</h1>
								<p>
									<a target="_blank" href="https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=0ahUKEwi6-Oy8yNLPAhVD1oMKHSUIDO4QFggcMAA&url=http%3A%2F%2Fscholar.google.com%2Fcitations%3Fuser%3D14dDyzkAAAAJ%26hl%3Den&usg=AFQjCNFTYlYtTgzRzLx8EcrkUANAWw86vg&sig2=K8-K_5yvLL4n1LZlCK2Wfw&bvm=bv.135258522,d.amc">Google Scholar</a>&nbsp;&nbsp;&nbsp;&nbsp;
									<a target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Jungbluth+SP">PubMed</a>&nbsp;&nbsp;&nbsp;&nbsp;
									<a target="_blank" href="http://orcid.org/0000-0001-9265-8341">ORCID</a>&nbsp;&nbsp;&nbsp;&nbsp;
									<a target="_blank" href="https://www.mendeley.com/profiles/sean-jungbluth/">Mendeley</a>&nbsp;&nbsp;&nbsp;&nbsp;
									<a target="_blank" href="http://www.researcherid.com/rid/A-9054-2012">ResearcherID</a>&nbsp;&nbsp;&nbsp;&nbsp;
								</p>
<!--         <h2>Publication Graphical Summary</h2>
        <p>For a full list of publications, see Google Scholar.</p>
								<center><img src="daily/sean_google-scholar-autosummary1.png" alt="" width="700" height="2800" border="5"></center><h1></h1>
								<center><img src="daily/sean_google-scholar-autosummary2a.png" alt="" width="700" height="2800" border="5"></center><h1></h1>
								<center><img src="daily/sean_google-scholar-autosummary2b.png" alt="" width="700" height="2800" border="5"></center><h1></h1>
								<center><img src="daily/sean_google-scholar-autosummary3.png" alt="" width="700" height="2800" border="5"></center><h1></h1>
								<center><img src="daily/sean_google-scholar-autosummary4.png" alt="" width="700" height="2800" border="5"></center><h1></h1>

<!-- -->
<!-- 								<h2>Published</h2>
								<p> Numbered list more or less based on date of original publication.</p>
								<p> <b>19)</b> McVeigh D, Skarke A, Dekas AE, Borrelli C, Hong W-L, Marlow J, Pasulka A, <b>Jungbluth SP</b>, Barco RA, Djurhuus A. Characterization of benthic biogeochemistry and ecology at three methane seep sites on the Northern US Atlantic Margin. Deep-Sea Research Part II.
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.dsr2.2018.03.001"></span><span><a target="_blank" href="docs/blank.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>18)</b> Anantharaman K, Hausmann B, <b>Jungbluth SP</b>, Kantor RS, Lavy A, Warren LA, Rappé MS, Pester M, Loy A, Thomas BC, Banfield JF. Expanded diversity of microbial groups that shape the dissimilatory sulfur cycle. ISMEJ.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/s41396-018-0078-0"></span><span><a target="_blank" href="docs/Anantharaman-ISME2018.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>17)</b> Teehara KB, <b>Jungbluth SP</b>, Acosta TE, Hellebrand E, Misra AK, Onac PB, Pflitsch A, Smith SM, Rappé MS, Telus M, Schorghofer N. (2018) Cryogenic minerals in Hawaiian lava tubes: A geochemical and microbiological survey. Geomicrobiology Journal 35: 227-241.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1080/01490451.2017.1362079"></span><span><a target="_blank" href="docs/Geomicrobiology-Journal_Ice-Cave-Accepted.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>16)</b> Bowers RM, Kyrpides NC, Stepanauskas R, Harmon-Smith M, Doud D, Reddy TBK, Schulz F, Jarett J, Rivers AR, Eloe-Fadrosh EA, Tringe SG, Ivanova NN, Copeland A, Clum A, Becraft ED, Malmstrom RR, Birren B, Podar M, Bork P, Weinstock GM, Garrity GM, Dodsworth JA, Yooseph S, Sutton G, Glöckner FO, Gilbert JA, Nelson WC, Hallam SJ, <b>Jungbluth SP</b>, Ettema TJG, Tighe S, Konstantinidis KT, Liu W-T, Baker BJ, Rattei T, Eisen JA, Hedlund B, McMahon KD, Fierer N, Knight R, Finn R, Cochrane G, Karsch-Mizrachi I, Tyson GW, Rinke C, The Genome Standards Consortium, Lapidus A, Meyer F, Yilmaz P, Parks DH, Eren AM, Schriml L, Banfield JF, Hugenholtz P, Woyke T. (2017) Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea. Nature Biotechnology 35: 725-731.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/nbt.3893"></span><span><a target="_blank" href="docs/nbt.3893-combined.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>15)</b> Momper L, <b>Jungbluth SP</b>, Lee M, Amend JP. (2017) Energy and carbon metabolisms in a deep terrestrial subsurface fluid microbial community. The ISME Journal 11: 2319-2333.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/ismej.2017.94"></span><span><a target="_blank" href="docs/ismej201794a.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>14)</b> Marlow J, Borrelli C, <b>Jungbluth SP</b>, Hoffman C, Marlow H, Girguis P, AT-36 Team. (2017) Telepresence is a potentially transformative tool for field science. Proceedings of the National Academy of Sciences USA 114: 4841-4844.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1073/pnas.1703514114"></span><span><a target="_blank" href="docs/PNAS-2017-Marlow-4841-4.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>13)</b> <b>Jungbluth SP</b>, Glavina del Rio T, Tringe SG, Stepanauskas R, Rappé MS. (2017) Genomic comparisons of a bacterial lineage that inhabits both marine and terrestrial deep subsurface systems. PeerJ 5:e3134.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.7717/peerj.3134"></span><span><a target="_blank" href="docs/peerj-3134.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>12)</b> <b>Jungbluth SP</b>, Amend JP, Rappé MS. (2017) Metagenome sequencing and 98 microbial genomes from Juan de Fuca Ridge flank subsurface fluids. Scientific Data 4:170037.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/sdata.2017.37"></span><span><a target="_blank" href="docs/sdata201737.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>11)</b> Nigro OD, <b>Jungbluth SP</b>, Lin H-T, Hsieh C-C, Miranda J, Schvarcz C, Rappé MS, Steward GF. (2017) Viruses in the oceanic basement. mBio 8: e02129-16.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1128/mBio.02129-16"></span><span><a target="_blank" href="docs/mBio-2017-Nigro.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>10)</b> Robador A, LaRowe, DE, <b>Jungbluth SP</b>, Lin H-T, Rappé MS, Nealson KH, Amend JP (2016) Nanocalorimetric characterization of microbial activity in deep subsurface oceanic crustal fluids. Frontiers in Microbiology 7: 454.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.3389/fmicb.2016.00454"></span><span><a target="_blank" href="docs/fmicb-07-00454.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>9)</b> <b>Jungbluth SP</b>, Bowers RM, Lin H-T, Cowen JP, Rappé MS. (2016) A novel microbial assemblage inhabits fluids within mid-ocean ridge flank subsurface basalt. The ISME Journal 10: 2033-2047.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/ismej.2015.248"></span><span><a target="_blank" href="docs/ISMEJ2016-merged.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>8)</b> Robador A, <b>Jungbluth SP</b>, LaRowe DE, Bowers RM, Amend JP, Rappé MS, Cowen JP. (2015) Activity and phylogenetic diversity of sulfate-reducing microorganisms in low-temperature subsurface fluids within the upper oceanic crust. Frontiers in Microbiology 5: 748.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.3389/fmicb.2014.00748"></span><span><a target="_blank" href="docs/fmicb-05-00119.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>7)</b> Biddle JF, <b>Jungbluth SP</b>, Lever MA, Rappé MS. (2014) Life in the ocean crust. In: Kallmeyer J (ed). Life in Extreme Environments: Microbial Life of the Deep Biosphere. DeGruyter. [Book Chapter; No Peer-Review]<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1515/9783110300130.29"></span><span><a target="_blank" href="docs/10.1515_9783110300130.29.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>6)</b> Böttjer D, <b>Jungbluth SP</b>, Boiteau R, Burkhardt B, De Leo F, Bruno BC. (2014) Career choices in marine and environmental sciences: navigating a sea of options. Oceanography 27(4): 37-43. [NON PEER-REVIEWED]<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.5670/oceanog.2014.35"></span><span><a target="_blank" href="docs/27-2_bottjer.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>5)</b> Lin H-T, Cowen JP, Olson EJ, Lilley MD, <b>Jungbluth SP</b>, Wilson ST, Rappé MS. (2014) Dissolved hydrogen and methane in the oceanic basaltic biosphere. Earth and Planetary Science Letters 405: 62-73.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1016/j.epsl.2014.07.037"></span><span><a target="_blank" href="docs/lin2014.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>4)</b> <b>Jungbluth SP</b>, Lin H-T, Cowen JP, Glazer BT, Rappé MS. (2014) Phylogenetic diversity of microorganisms in subseafloor crustal fluids from boreholes 1025C and 1026B along the Juan de Fuca Ridge flank. Frontiers in Microbiology 5: 119.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.3389/fmicb.2014.00119"></span><span><a target="_blank" href="docs/fmicb-05-00119.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>3)</b> <b>Jungbluth SP</b>, Johnson LGH, Cowen JP, Rappé MS. (2013). Data report: microbial diversity in sediment near Grizzly Bare Seamount from Holes U1363B and U1363G. Proceedings of the Integrated Ocean Drilling Program 327: pp. 1-27.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.2204/iodp.proc.327.201.2013"></span><span><a target="_blank" href="docs/327_201.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>2)</b> <b>Jungbluth SP</b>, Grote J, Lin H-T, Cowen JP, Rappé MS. (2013). Microbial diversity within basement fluids of the sediment-buried Juan de Fuca Ridge flank. The ISME Journal 7(1): 161-172.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1038/ismej.2012.73"></span><span><a target="_blank" href="docs/jungbluth_ISMEJ2013_full.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>1)</b> Fisher AT, Tsuji T, Petronotis K, Wheat CG, Becker K, Clark JF, Cowen J, Edwards K, Jannasch H, IODP Expedition 327 and <b>Atlantis Expedition AT18-07 Shipboard Parties</b> (2012). IODP Expedition 327 and Atlantis Expedition AT18-07: Observatories and Experiments on the Eastern Flank of the Juan de Fuca Ridge. Scientific Drilling 13: 4-11.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.2204/iodp.sd.13.01.2011"></span><span><a target="_blank" href="docs/sd-13-4-2012.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<h2>Published PrePrints</h2>
								<p> Anatharaman K, <b>Jungbluth SP</b>, Kantor RS, Lavy A, Warren LA, Rappé MS, Thomas BC, Banfield JF. Dramatic expansion of microbial groups that shape the global sulfur cycle. 166447 bioRxiv.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.1101/166447"></span><span><a target="_blank" href="docs/166447.full.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>Jungbluth SP</b>, Amend JP, Rappé MS. (2016) Metagenome sequencing and 98 microbial genomes from Juan de Fuca Ridge flank subsurface fluids. PeerJ PrePrints.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.7287/peerj.preprints.2613v1"></span><span><a target="_blank" href="docs/peerj-preprints-2613.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<p> <b>Jungbluth SP</b>, Glavina del Rio T, Tringe SG, Stepanauskas R, Rappé MS. (2016) Genomic comparisons of a bacterial lineage that inhabits both marine and terrestrial deep subsurface systems. PeerJ PrePrints.<br>
								<div align="right"><span class='altmetric-embed' data-badge-type='donut' data-doi="10.7287/peerj.preprints.2592v1"></span><span><a target="_blank" href="docs/peerj-preprints-2592.pdf"><img src="images/button.png" alt="download pdf" width="60" height="25"></a></span></div></p>
								<h2>Submitted/In Revison/Accepted/In Press</h2>
								<p> None right now!</p>
								<h2>In Prep</h2>
								<p> <b>Jungbluth SP</b>, Paver S, Carreira C, Fong A, Maier M, Tolar B, Williams A. Toward predictive microbial ecology: how we do, don’t, and should study aquatic microbial communities. Submitted.<br>
								<p> <b>Jungbluth SP</b>, Chadwick G, Orphan V, Amend JP, Rappé MS. A novel methanogenic Archaea. In prep.<br>
								<p> Robador A, <b>Jungbluth SP</b>, LaRowe DE, Amend JP. Invited review: Microbial activity in deep subseafloor crustal fluids. In prep for Frontiers in Microbiology. (Anticipated submission: Q3, 2017)<br> -->
						 </div>
						</section><!-- /.publications-text -->
						<section id="consulting-text" class="inactive">
							<div class="col-sm-12 col-md-12">
								<h1>Consulting</h1>
								<p>Effective computational biology requires on-the-fly thinking rooting in a combined understanding of biology and scripting. That's where my knowledge of microbiology, molecular biology, genomics, informatics, databases, and software development can help! Bioinformatic consultation services via my J1ab Professional Consulting service can help you construct and run fully-customized informatic pipelines. </p>
								<h1></h1>
								<p>Academic and commercial clients are provided consultation for an hourly or fixed price fee. Rates are negotiable and are on a case-by-case basis. Please contact me if interested. </p>
							</div>
						</section><!-- /.consulting-text -->
						<section id="softwarepublished-text" class="inactive">
							<div class="col-sm-12 col-md-12">
								<h1>Published Software</h1>
								<p>Links correspond to various software and tools I've written and published online.</p>
								<h2>Applications</h2>
								<p><a target="_blank" href="https://github.com/jungbluth/kb_iMAG-viz"><b>kb_iMAG-viz</b></a></p>
								<p>kb_iMAG-viz is a KBase implementation of the iMAG-viz pipeline. iMAG-viz is a tool designed for quality-checking microbial genome data. Stay tuned for a more complete description and working demo.</p>
								<center><img src="public-images/kbase-org-logo.png" alt="under-construction" width="225" height="225" border="5"></center>
								<h2>Tools/Utilities</h2>
								<p><a target="_blank" href="https://github.com/jungbluth/git-commit-timeline"><b>git-commit-timeline</b></a></p>
								<p>Git Commit Timeline is a Shell utility to show your Github commit history as a series of histogram plots separated by repository. I wrote this one weekend while taking a deep dive into the nuances of Shell scripting.</p>
								<center><img src="images/git-commit-timeline.png" alt="git-commit-timeline" width="212" height="290" border="5"></center>
								<h3></h3>
								<p><a target="_blank" href="https://github.com/jungbluth/kb_sdk_icon_generator"><b>kb_sdk_icon_generator</b></a></p>
								<p>kb_sdk_icon_generator is a tool for auto-generation of icons for KBase SDK Apps. I wrote this to speed up the process of generating files/content for these applications.</p>
								<center><img src="images/kb_sdk_icon_generator.jpg" alt="kb_sdk_icon_generator" width="400" height="133" border="5"></center>
								<h3></h3>
								<p><a target="_blank" href="https://github.com/jungbluth/scholar-annotator"><b>scholar-annotator (unpublished; under development)</b></a></p>
								<p>scholar-annotator is an Rscript to, through the use of visualization, produce a narrative to accompany one's Google Scholar publication record. I wrote this because I was tired of updating my personal website to reflect newest publications, etc., and further, I thought that the information and convenient data structures provided by Google Scholar were a relatively untapped opportunity to explore further.</p>
							 <center><img src="images/scholar-annotator.jpg" alt="scholar-annotator" width="420" height="290" border="5"></center>
								<h3></h3>
								<p><a target="_blank" href="https://github.com/jungbluth/KBase-app-navigator"><b>KBase-app-navigator (unpublished; under development)</b></a></p>
								<p>KBase-app-navigator presents a way to visualize the relatedness of KBase Apps based on the required input and output data types. I wrote this application because I found it challenging to sense the available KBase Apps for distinct analysis workflows. My current plan is to develop this further before releasing.</p>
							 <center><img src="images/kbase-app-navigator-output.png" alt="kbase-app-navigator" width="420" height="290" border="5"></center>
							</div>
						</section><!-- /.consulting-text -->
<!-- 						<section id="field-text" class="inactive">
							<div class="col-sm-12 col-md-12">
							</div>
						</section><!-- /.field-text --> -->
						<section id="data-text" class="inactive">
							<div class="col-sm-12 col-md-12 col-lg-12">
								<h1>Data</h1>
								<h2>Subseafloor Biosphere</h2><h1></h1><h1></h1>
								<h3>BCO-DMO</h3>
								<p><a target="_blank" href="http://www.bco-dmo.org/dataset/660489">Link to BCO-DMO Data Entree</a></p>
								<p><a rel="nofollow" href="docs/BCO-DMO-all-data.xlsx" target="_blank">Download BCO-DMO Data (.xlsx)</a></p>
								<p>The BCO-DMO link will help you access data related to my publications. Please let me know if you can't find some data or information that you need.</p><h1></h1><h1></h1>
								<h3>Jungbluth et al. 2013 <i>The ISME Journal</i></h3>
								<p><a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/?term=JX194240%3AJX194738%5Baccn%5D+OR+JX255729%3AJX255730%5Baccn%5D">NCBI Genbank Sequences (n=501)</a></p>
								<p>This link takes you to the SSU rRNA sequence data from borehole crustal fluid and seawater microbial communities. These sequences were published with the paper.</p>
								<p><a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/?term=KP721993%3AKP722019%5Baccn%5D">NCBI Genbank Sequences (n=27) from unpublished data.</a></p>
								<p>This link takes you to the SSU rRNA sequence data from borehole crustal fluid microbial communities. These data are unpublished but generated with different gene primers from an identical sample used in the publication.</p><h1></h1><h1></h1>
								<h3>Jungbluth et al. 2013 <i>Proceedings of the Integrated Ocean Drilling Program</i></h3>
								<p><a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/?term=KC990940%3AKC991019%5Baccn%5D">NCBI Genbank Sequences (n=80)</a></p>
								<p>This link takes you to the SSU rRNA sequence data from sediment microbial communities.</p><h1></h1><h1></h1>
								<h3>Jungbluth et al. 2014 <i>Frontiers in Microbiology</i></h3>
								<p><a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/?term=KF574286%3AKF574384%5Baccn%5D">NCBI Genbank Sequences (n=99)</a></p>
								<p>This link takes you to the SSU rRNA sequence data from borehole crustal fluid microbial communities.</p><h1></h1><h1></h1>
								<h3>Robador et al. 2015 <i>Frontiers in Microbiology</i></h3>
								<p><a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/?term=KP118849%3AKP118879%5Baccn%5D">NCBI Genbank Sequences (n=31)</a></p>
								<p>This link takes you to the dsrAB sequence data from borehole crustal fluid microbial communities.</p>
								<p><a target="_blank" href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA266850">NCBI BioProject entree</a></p>
								<p>This link takes you to the SSU rRNA amplicon sequence data from borehole crustal fluid microbial communities.</p><h1></h1><h1></h1>
								<h3>Jungbluth et al. 2016 <i>The ISME Journal</i></h3>
								<p><a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/?term=KR072702%3AKR072893%5Baccn%5D">NCBI Genbank Sequences (n=192)</a></p>
								<p>This link takes you to the SSU rRNA sequence data from borehole crustal fluid, sediment, and seawater microbial communities.</p>
								<p><a target="_blank" href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA266365">NCBI BioProject entree</a></p>
								<p>This link takes you to the SSU rRNA amplicon sequence data from borehole crustal fluid, sediment, and seawater microbial communities.</p><h1></h1><h1></h1>
								<h3>Jungbluth et al. 2017 <i>Scientific Data</i></h3>
								<p><a target="_blank" href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA269163">NCBI BioProject entree for samples used for metagenomics.</a></p>
								<p><a target="_blank" href="https://img.jgi.doe.gov/cgi-bin/mer/main.cgi?section=TaxonDetail&page=taxonDetail&taxon_oid=3300002481">IMG/MER sample entree for U1362A metagenome data.</a></p>
								<p><a target="_blank" href="https://img.jgi.doe.gov/cgi-bin/mer/main.cgi?section=TaxonDetail&page=taxonDetail&taxon_oid=3300002532">IMG/MER sample entree for U1362B metagenome data.</a></p>
								<p><a target="_blank" href="https://figshare.com/articles/Genome-Bin_JGI-Scaffold-Files_zip/4269590">Files needed to isolate scaffold sets for all 98 metagenome-assembled genomes (MAGs) in IMG/M.</a></p>
								<p><a target="_blank" href="https://figshare.com/articles/Genome-Bin_Fasta-Files_zip/4269587">FASTA files containing the contigs of all 98 metagenome-assembled genomes (MAGs).</a></p>
								<p><a target="_blank" href="https://figshare.com/articles/SSU-rRNA-genes-from-bins-gt300bp_fasta/4269593">FASTA file containing 54 SSU rRNA genes with length >300 base pairs extracted from the 98 metagenome-assembled genomes (MAGs).</a></p>
								<p><a target="_blank" href="https://figshare.com/articles/Full-IMG-Bin-Annotation-Info_txt/4269581">IMG/M annotations associated with the scaffolds of all 98 metagenome-assembled genomes (MAGs).</a></p>
								<h3>Jungbluth et al. 2017 <i>PeerJ</i></h3>
								<p><a target="_blank" href="https://figshare.com/articles/EMIRGE-reconstructed-SSU-rRNA-genes-from-1362A-1362B-metagenomes_fasta/4539149">EMIRGE reconstructed SSU rRNA genes from U1362A and U1362B metagenome data.</a></p>
								<h2>Basaltic Ice Cave</h2><h1></h1><h1></h1>
								<h3>Unpublished data</h3>
								<p><a target="_blank" href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA340160">NCBI BioProject entree unpublished samples used for SSU rRNA amplicon sequencing.</a></p>
								<p>This link takes you to the SSU rRNA amplicon sequence data from Hawaii ice cave microbial communities.</p>
							</div>
						</section><!-- /.data-text -->
<!-- 						<section id="contact-text" class="inactive">
							<div class="col-sm-12 col-md-12 col-lg-12">
								<h1>Contact</h1>
								<h2></h2>
								<p> jungbluth dot sean at gmail dot com</p>
							</div>
						</section> --><!-- /.contact-text -->
					</div><!-- /.templatemo-content -->
				</div><!-- /.templatemo-content-wrapper -->
			</div><!-- /.row -->
			<div class="row">
				<div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 footer">
					<p class="footer-text">
						<span>Copyright &copy; 2017-2022 Sean P. Jungbluth</span>
					</p>
				</div><!-- /.footer -->
			</div>
	</div><!-- /#main-wrapper -->
	<div id="preloader">
		<div id="status3">&nbsp;</div>
	</div><!-- /#preloader -->
	</body>
</html>
